synfig-studio/src/gui/preview.cpp

```diff
@@ -464,7 +464,7 @@ Widget_Preview::Widget_Preview():
 
 	//loop
 	button = &b_loop;
-	IMAGIFY_BUTTON(button,"synfig-animate_loop", _("Loop"));
+	IMAGIFY_BUTTON(button,"synfig-animate_loop", _("Repeat"));
```

Comment:
For me, "repeat" doesn't have the same meaning of "loop".

Reply:
I thought about that too, maybe changing the tooltip used for canvas to Loop is better?

Reply:
![repeat](https://user-images.githubusercontent.com/6705690/102941181-540df380-44b2-11eb-8d61-ad1f98beec28.png)

Reply:
Done!